### ISSUE TITLE
Event constructor doesn't support composed in compat mode

### ISSUE THREAD

**Issue by pmdartus** [salesforce/lwc#1139]
### Description

When running in compat mode, standard `Event` constructor doesn't accept `composed` in option bag.

```js
const evt = new Event('test', { composed: true });
console.log(evt.composed);
```

#### Expected Results
```
true
```

#### Actual Results
```
false
```

### Browsers Affected
IE11

### Version
- LWC: 0.37.0

**Comment by ravijayaramappa** [comment]
GUS work item: W-5983676

**Comment by pmdartus** [comment]
Fixed with #1676
